<launch>

  <node pkg="loam_interface" type="loamInterface" name="loamInterface" output="screen" required="true">
    <!-- <param name="stateEstimationTopic" type="string" value="/integrated_to_init" /> -->
    <param name="stateEstimationTopic" type="string" value="/Odometry" />
    <!-- <param name="stateEstimationTopic" type="string" value="/livox_odometry_mapped" /> -->
    <param name="registeredScanTopic" type="string" value="/livox_pcl" />
    <!-- <param name="registeredScanTopic" type="string" value="/livox_full_cloud" /> -->
    <param name="flipStateEstimation" type="bool" value="false" />
    <param name="flipRegisteredScan" type="bool" value="false" />
    <param name="sendTF" type="bool" value="true" />
    <param name="reverseTF" type="bool" value="false" />
  </node>

</launch>
